src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -171,6 +183,36 @@ public void undoListModification() {
         public void updateSortedPersonList(Comparator<Person> comparator) {
             throw new AssertionError("This method should not be called.");
         }
+
+        @Override
+        public void setShortcutLibrary(ShortcutLibrary shortcutLibrary) {
+
+        }
+
+        @Override
+        public ShortcutLibrary getShortcutLibrary() {
+            return null;
+        }
+
+        @Override
+        public boolean hasShortcut(String shortcutName) {
+            return false;
+        }
+
+        @Override
+        public void deleteShortcut(String shortcutName) {
+
+        }
+
+        @Override
+        public void addShortcut(String shortcutName, String shortcutCommand) {
+
+        }
+
+        @Override
+        public void setShortcut(String target, String shortcutCommand) {
+
+        }
```

Comment:
Is there a better alternative to having these methods empty? If they are not going to be called, perhaps you could throw an assertionError? If not it is also fine if it doesn't affect the test cases.

Reply:
I have inserted assertion errors in the methods you mentioned. Thanks!